griptape/config/openai_structure_config.py

```diff
@@ -20,37 +11,24 @@
 
 
 @define
-class OpenAiStructureConfig(BaseStructureConfig):
-    global_drivers: StructureGlobalDriversConfig = field(
-        default=Factory(
-            lambda: StructureGlobalDriversConfig(
-                prompt_driver=OpenAiChatPromptDriver(model="gpt-4o"),
-                image_generation_driver=OpenAiImageGenerationDriver(model="dall-e-2", image_size="512x512"),
-                image_query_driver=OpenAiVisionImageQueryDriver(model="gpt-4-vision-preview"),
-                embedding_driver=OpenAiEmbeddingDriver(model="text-embedding-3-small"),
-                vector_store_driver=LocalVectorStoreDriver(
-                    embedding_driver=OpenAiEmbeddingDriver(model="text-embedding-3-small")
-                ),
-            )
-        ),
-        kw_only=True,
+class OpenAiStructureConfig(StructureConfig):
+    prompt_driver: OpenAiChatPromptDriver = field(
+        default=Factory(lambda: OpenAiChatPromptDriver(model="gpt-4o")), metadata={"serializable": True}
+    )
```

Comment:
Should this (and similar fields) have `kw_only=True`?

Reply:
Yes good catch